.travis.yml

```diff
@@ -14,7 +14,3 @@ notifications:
   email:
     on_success: change
     on_failure: change
-branches:
-  only:
-    - master
-    - generate-query-models
```

Comment:
JFYI: This was done to speed up CI, which I guess that's not a problem for you ;)